fedocal/fedocallib/__init__.py

```diff
@@ -722,6 +737,8 @@ def add_meetings_to_vcal(ical, meetings):
         be added.
     :arg meetings: a list of fedocal.model.Meeting object to convert to
         iCal and add to the provided calendar.
+    :kwarg enable_reminder: a boolean telling whether to add reminder
+        for the meetings in the iCal object or not.
```

Comment:
hm, what is ``enable_reminder`` used for in this method, did you forget a piece? :)